supply/lib/supply/client.rb

```diff
@@ -78,8 +78,20 @@ def initialize(service_account_json: nil, params: nil)
 
     def call_google_api
       yield if block_given?
-    rescue Google::Apis::ClientError => e
-      UI.user_error!("Google Api Error: #{e.message}")
+    rescue Google::Apis::Error => e
+      error = begin
```

Comment:
would have preferred this to be an inline `resuce nil`, but rubocop didn't like it.